docs/source/guide/general/keys.rst

```diff
@@ -0,0 +1,77 @@
+############
+Singing Keys
+############
+
+Repository Service for TUF (RSTUF) requires two sets of keys for
+:ref:`guide/deployment/index:Deployment` and
+:ref:`guide/deployment/setup:Service Setup`:
+:ref:`guide/general/keys:Root Key(s) (offline)` and the
+:ref:`guide/general/keys:Online Key`.
+
+
+Root Key(s) (offline)
+#####################
+
+The Root key(s) delegates trust to TUF. The number of keys is
+the number of identities/people who administrate the top-level TUF Metadata.
+
+RSTUF requires all Root key(s) only during the
+:ref:`guide/deployment/setup:Service Setup` specifically during the
+:ref:`guide/deployment/setup:Ceremony` process. This process also defines the
+Root key threshold, representing the number of Root key(s) for future offline
+operations.
+
+.. note::
+  .. collapse:: See the number of Root keys/threshold example
+
+      An organization defines that it will have 5 (five) Root keys to
+      administrate the RSTUF Service. A total of 5 (five) people will require
+      to use their keys individually during the Ceremony process.
```

Comment:
```suggestion
      An organization declares that it will utilize 5 (five) Root keys in order to
      administer the RSTUF Service. All 5 (five) people will be required
      to utilize their keys individually during the Ceremony process.
```

Reply:
So, I'm not sure if all five people will need to participate in the Ceremony, or they simply have the option of participating. Perhaps you need to explain this to me.

Reply:
All five people will need to participate in the ceremony once they all need to load their keys.
We are working on a feature where they can participate in this ceremony asynchronously.